### PR TITLE
Update x11-terms/sakura to 3.6.0

### PR DESCRIPTION
**Pull request by kennyballou**
- Add new ebuild for `x11-libs/vte-0.50.4`
- Add new ebuild for `x11-terms/sakura`.

Bug: https://bugs.gentoo.org/657256
Bug: https://bugs.gentoo.org/654834

### CODE REVIEW

File: x11-terms/sakura/sakura-3.6.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+	${RDEPEND}
+	>=dev-lang/perl-5.10.1
+	virtual/pkgconfig
+"
+
+PATCHES=(
+	"${FILESDIR}"/${PN}-3.1.3-flags.patch
+)
+
+DOCS=(
```
Comment (Zlogene) [inline x11-terms/sakura/sakura-3.6.0.ebuild]: it should be installed by default

File: x11-terms/sakura/sakura-3.6.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,66 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit cmake-utils eutils flag-o-matic gnome2-utils xdg-utils
+
+DESCRIPTION="sakura is a terminal emulator based on GTK and VTE"
```
Comment (Zlogene) [inline x11-terms/sakura/sakura-3.6.0.ebuild]: `DESCRIPTION` should not contain a package name

File: x11-terms/sakura/sakura-3.6.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+HOMEPAGE="http://www.pleyades.net/david/projects/sakura/"
+SRC_URI="https://launchpad.net/${PN}/trunk/${PV}/+download/${P}.tar.bz2"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="amd64 x86 ~arm-linux ~x86-linux"
```
Comment (Zlogene) [inline x11-terms/sakura/sakura-3.6.0.ebuild]: do not commit straight to  stable, it should be `~arch` for every keyword

**Comment by kennyballou** [comment]
@Zlogene: I've updated the PR. Please let me know if there is anything else that is needed for this change.

**Comment by a17r** [comment]
Meanwhile, newer versions are in Gentoo ebuild repo.

**Pull request closed without merging**